src/Orleans.Core.Abstractions/IDs/SystemTargetGrainId.cs

```diff
@@ -14,20 +14,19 @@ namespace Orleans.Runtime
     public readonly struct SystemTargetGrainId : IEquatable<SystemTargetGrainId>, IComparable<SystemTargetGrainId>, ISpanFormattable
     {
         private const char SegmentSeparator = '+';
-        private readonly GrainId _grainId;
```

Comment:
This should be reverted - it's stored as a field specifically to get better inlining (when accessing it through a property, the subfields cannot be directly addressed by the compiler unless the entire data structure is "inlined").